igc_lib_demo.py

```diff
@@ -7,11 +7,13 @@
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 2:
-        print "Please pass an .igc file in argv"
-        sys.exit(1)
+    if len(sys.argv) < 2:
+        print "Usage: %s file.igc [file.lkt]" % sys.argv[0]
```

Comment:
Missing `sys.exit(1)`, now the code will print the message and happily proceed :)